Crashing when no phrase is passed

When no phrase is passed and the message isn't a reply, the bot crashes.

```
Environment: Docker
Bot version: 1.0.3
Self-hosted: ❌ 
```

<details>
<summary>Logs</summary>
panic: runtime error: index out of range [2] with length 2
goroutine 2989 [running]:
github.com/lucxjo/diru/cmd.Dpl(0xc00000c3c0, {0x7de4c0, 0xc00010cc60}, 0x1)
	/app/cmd/dpl.go:14 +0x305
github.com/lucxjo/diru/cmd.Commands(0xc00000c3c0, {0x7de4c0, 0xc00010cc60}, 0xc00001c200, {{0xc000116300, 0x3b}, {0xc00006e4b0, 0x24}, {{0xc000150140, 0x9b}, ...}})
	/app/cmd/commands.go:19 +0xa7
main.main.func2({0x7de4c0, 0xc00010cc60}, 0x20)
	/app/diru.go:39 +0x65
github.com/andersfylling/disgord.(*dispatcher).trigger(0xc0004ae0c0, {0x6f6080, 0xc0004ac050}, {0x71c1c0, 0xc0004046a0})
	/go/pkg/mod/github.com/andersfylling/disgord@v0.32.3/reactor_gen.go:621 +0x477
github.com/andersfylling/disgord.(*dispatcher).dispatch(0xc0000543c0, {0xc00001a970, 0xe}, {0x71c1c0, 0xc0004046a0})
	/go/pkg/mod/github.com/andersfylling/disgord@v0.32.3/reactor.go:148 +0x5ae
created by github.com/andersfylling/disgord.(*Client).demultiplexer
</details>

--- a/cmd/commands.go
+++ b/cmd/commands.go
@@ -20,7 +20,7 @@ func Commands(msg *disgord.Message, s disgord.Session, c *deepl.Client, config c
 	} else if prefix == "dpla" {
 		Dpla(msg, s, c)
 	} else if prefix == "info" {
-		msg.Reply(context.Background(), s, "Diru is a Discord bot that can translate text.\n\n**Technical information:**\n```" + "OS: " + runtime.GOOS + "\n" + "Arch: " + runtime.GOARCH + "\n" + "Go Version: " + runtime.Version()+"\n" + "Version: 1.0.2" + "\n" + "Source: https://github.com/Lucxjo/Diru/```")
+		msg.Reply(context.Background(), s, "Diru is a Discord bot that can translate text.\n\n**Technical information:**\n```" + "OS: " + runtime.GOOS + "\n" + "Arch: " + runtime.GOARCH + "\n" + "Go Version: " + runtime.Version()+"\n" + "Version: 1.0.4" + "\n" + "Source: https://github.com/Lucxjo/Diru/```")
 	} else if prefix == "issue" {
 		msg.Reply(context.Background(), s, "Please report any issues on the GitHub issue tracker: https://github.com/Lucxjo/Diru/issues")
 	} else if prefix == "vote" && config.Topgg.Token != "" && config.Topgg.Id != "" {

--- a/cmd/dpl.go
+++ b/cmd/dpl.go
@@ -11,7 +11,14 @@ import (
 
 // Dpl uses the Translate function to translate text to a requested language.
 func Dpl(msg *disgord.Message, s disgord.Session, c *deeplgo.Client) {
-	lang := strings.Split(msg.Content, " ")[2]
+	sArr := strings.Split(msg.Content, " ")
+
+	if len(sArr) <= 2 && msg.Type != disgord.MessageTypeReply {
+		msg.Reply(context.Background(), s, "Please specify a language code.\n Please use the format `@Diru dpl <lang> <phrase>`.\n For language codes see: https://github.com/Lucxjo/Diru/wiki/Supported-translators-and-languages#deepls-supported-languages")
+		return
+	}
+
+	lang := sArr[2]
 
 	if !deepl.CheckCode(lang) {
 		msg.Reply(context.Background(), s, "Language code is incorrect.\n Please use the format `@Diru dpl <lang> <phrase>`.\n For language codes see: https://github.com/Lucxjo/Diru/wiki/Supported-translators-and-languages#deepls-supported-languages")

